src/jobs/taskcluster_graph.js

```diff
@@ -84,7 +84,11 @@ export default class TaskclusterGraphJob extends Base {
     let pushdate = push.date;
 
     let level = projectConfig.level(this.config.try, repo.alias);
-    let scopes = projectConfig.scopes(this.config.try, repo.alias)
+    let scopes = projectConfig.scopes(this.config.try, repo.alias);
+
+    // All pushes should at least be able to send
+    // emails to the submitting user if desired.
+    scopes.push('route.notify.email.' + repo.user + '.*');
```

Comment:
I made this change at the level of `projectConfig` itself in the latest diff. I figured I would do this because I think we actually want the config to be immutable anywhere that imports the config. If that's not true, I can amend the commit to just `slice()` here instead. Either way is fine with me.